aries-site/src/examples/templates/dashboards/components/Measure.js

```diff
@@ -0,0 +1,34 @@
+import { cloneElement } from 'react';
+import PropTypes from 'prop-types';
+import { Box, NameValuePair, Text } from 'grommet';
+
+export const Measure = ({ name, value, onClick }) => {
+  const { icon, label } = name;
+  const adjIcon = cloneElement(icon, { size: 'small' });
+
+  return (
+    <Box onClick={onClick}>
```

Comment:
Should this be `Button`? I don't think Box with onClick currently renders as `button` in the DOM and I'm wondering if it would be better for accessibility to have this render as `button`.

Reply:
Done.